src/LoginError.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+function LoginError({onLogin}){
+    return(
+      <main>
+        <h1>Login Unsuccessful!</h1>
+        <p><span id='username'>Hey Student!</span> there was an error with your attendance! Please try again!</p>
```

Comment:
Wrap around 'Student'.
```suggestion
        <p>Hey <span id='username'>Student</span>! There was an error with your attendance. Please try again!</p>
```